doc/source/rllib.rst

```diff
@@ -171,15 +171,18 @@ Custom Environments
 To train against a custom environment, i.e. one not in the gym catalog, you
 can register a function that creates the env to refer to it by name. The contents of the
 ``env_config`` agent config field will be passed to that function to allow the
-environment to be configured. For example:
+environment to be configured. The return type should be an `OpenAI gym.Env <https://github.com/openai/gym/blob/master/gym/core.py>`__. For example:
 
 ::
 
     import ray
     from ray.tune.registry import register_env
     from ray.rllib import ppo
 
-    env_creator = lambda env_config: MyCustomEnv(env_config)
+    def env_creator(env_config):
```

Comment:
for this code block (and any other python clodeblocks), can you replace `::` with `.. code-block:: python` to get the syntax highlighting? For code intended to run on the command line, we can use `.. code-block:: bash`.

Reply:
Done